hr_holidays_compute_days/models/hr_holidays.py

```diff
@@ -99,24 +99,26 @@ def _inverse_date_from_full(self):
         as fallback.
         """
         for record in self.filtered('from_full_day'):
-            tz_name = record.employee_id.user_id.tz or record.env.user.tz
-            dt = fields.Datetime.from_string(record.date_from_full).replace(
-                hour=0, minute=0, second=0, microsecond=0,
-                tzinfo=tz.gettz(tz_name),
-            ).astimezone(tz.tzutc())
-            record.date_from = fields.Datetime.to_string(dt)
+            if record.date_from_full:
```

Comment:
Instead of doing this big diff, you can just:
```python
if not record.date_from_full:
    continue
```

Reply:
I pushed the change like you suggested